components/dashboard/src/teams/Members.tsx

```diff
@@ -46,6 +47,10 @@ export default function() {
         })();
     }, [ team ]);
 
+    useEffect(() => {
+        setLeaveTeamEnabled(members.length > 1);
```

Comment:
Shouldn't it be filtered by team owners?

Reply:
absolutely! just updated